py/requirements.txt

```diff
@@ -1,9 +1,9 @@
 numpy
 packaging
 pybind11==2.6.2
---extra-index-url https://download.pytorch.org/whl/nightly/cu124
-torch>=2.5.0.dev,<2.6.0
-torchvision>=0.19.0.dev,<0.20.0
+--extra-index-url https://download.pytorch.org/whl/test/cu124
+torch==2.4.0
```

Comment:
This can be versions >=2.4.0,<2.5.0

Reply:
@gs-olive, @peri044 can we remove this requirements.txt since its in pyproject.toml?